Subscriptions/routes/membersRouter.ts

```diff
@@ -47,8 +47,10 @@ router.put("/:memberId", async (req, res, next) => {
 router.delete("/:memberId", async (req, res, next) => {
   try {
     const memberId = req.params.memberId;
```

Comment:
You can use the syntax of destructuring instead -> `const { memberId } = req.params;`
Usually if we are using a object param, that we need all of it's properties it will be easier to do it like that -> 
`const {param1, param2, param3} = req.params;`